### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by kev1n80**
This change reorganizes the content in the webpage and adds simple styling in order to offer a better user experience.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -9,34 +9,82 @@
...
+            <img src="images/luke_skywalker_figure.jpg" alt="Luke Skywalker using the force">
+                <p>Looking around my desk, I was inspired by the figurines that give my workspace
+                    character. When I looked at the fortune cookie I noticed how beautful 
+                    reflected light can be and the magical experience it gives. When I prepared
+                    to take a picture of the fortune cookie I noticed that the reflect light 
+                    in front of Luke Skywalker makes it seem like he is using the force.
+                </p>
+        </div>
+        <div class="photo-container">
+            <img src="images/sunset_coffee.jpg" alt="sunset in a cup of coffee">
```
Comment (saucepwn) [inline portfolio/src/main/webapp/index.html]: Thanks for using alt tags! These go a long way in helping people who have vision impairments use our products.

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -2,17 +2,74 @@
...
   margin-right: auto;
   width: 650px;
+  text-align: center;
 }
 
-#greeting-container {
-  margin-top: 20px;
+/* intro section */
+h1 {
+    margin-left: 30px;
```
Comment (saucepwn) [inline portfolio/src/main/webapp/style.css]: Ensure indentation is consistent throughout the file; this file started with a 2-space indent, I'd keep using that.

**Review by saucepwn — APPROVED** [review]

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -67,36 +67,29 @@ <h3>Photography</h3>
...
-            Looking around my desk, I was inspired by the figurines that give 
-            my workspace character. 
-            When I looked at the fortune cookie I noticed how beautful 
-            reflected light can be and the magical experience it gives. 
-            When I prepared to take a picture of the fortune cookie I noticed 
-            that the reflect light in front of Luke Skywalker makes it seem 
-            like he is using the force.
+            Looking around my desk, I was inspired by the figurines that give my workspace character. 
+            When I looked at the fortune cookie I noticed how beautful reflected light can be and the magical experience it gives. 
+            When I prepared to take a picture of the fortune cookie I noticed that the reflect light in front of Luke Skywalker makes it seem like he is using the force.
```
Comment (saucepwn) [inline portfolio/src/main/webapp/index.html]: Try to keep your lines to 100 characters or less to make it easier to view the entire line

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -73,3 +75,14 @@ h1 {
...
 }
+
+#blog-post-container {
+  text-align: left;
+  padding: 10px;
+  border: 2px black solid;
+  background-color: beige;
+}
+
+#blog-post-container p{
```
Comment (saucepwn) [inline portfolio/src/main/webapp/style.css]: nit: Add a space between "p{" -> "p {"

**Review by saucepwn — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (kev1n80) [reply portfolio/src/main/webapp/index.html]: Yup sorry about that, the line is shortened in the Google Cloud Shell editor automatically, but I have to indent it in order for it to show on the actual file.

**Pull request closed without merging**